### ISSUE TITLE
`?print-pdf` broken?

### ISSUE THREAD

**Issue by nilshg** [hakimel/reveal.js#1164]
There might be a problem with pdf printing, after switching to the latest master appending the `?print-pdf` query string makes half my slides disapper; [here's an example](http://webspace.qmul.ac.uk/ngudat/teaching/ECN223/Week7.html?print-pdf#/). 

I haven't changed the slides themselves, so I'm assuming this is a problem in pdf.css!?

**Comment by callahad** [comment]
Looks like you're still serving up the [reveal.min.js](http://webspace.qmul.ac.uk/ngudat/teaching/ECN223/js/reveal.min.js) from 2.6.1. Did you run the Grunt task to generate the reveal.min.js and reveal.min.css files?

**Comment by nilshg** [comment]
Excellent point, @callahad! For some reason, the `reveal.min.css` file wasn't swapped out when I uploaded the new reveal.js master to the server, it's working now. Thanks!

**Comment by nilshg** [comment]
Sorry, I was a bit quick in closing this. I actually ran into a few more issues:

1. I'm not sure I understand you correctly, but I believe your suggestion was to run `Gruntfile.js`? This doesn't work, and exits with an error.

2. Minifying the `reveal.css`

**Comment by callahad** [comment]
No problem, sorry for the too-short response :)

Basically, when you download the `master` version of reveal, you only get the basic source files. Others, like `reveal.min.css` and `reveal.min.js` can be derived from those base files, and need to be generated on your end if you'd like to use them.

To do that, you can use [Grunt](http://gruntjs.com/) to run "tasks" from [reveal.js's Gruntfile](https://github.com/hakimel/reveal.js/blob/8858ee75a5b5a74c4ed102555c7baaa0f5d29ecd/Gruntfile.js).

If your on Linux or Mac OS X and have [Node.js](https://nodejs.org/) installed, the quick way to do this is to `cd` into your reveal directory and run:

    npm install
    npm install grunt-cli
    ./node_modules/.bin/grunt

Which should, if all goes well, will build minified versions of those assets.

Alternatively, you could just edit your `Week7.html` to refer to `reveal.js` and `reveal.css` instead of `reaval.min.js` and `reveal.min.css` respectively. :)

**Comment by callahad** [comment]
Hm, the background being black appears to be a bug (it shows up on the demo `index.html` shipped with Reveal). You could switch in the default theme for a temporary fix, at the cost of visual consistency with your other weeks.

**Comment by callahad** [comment]
Hah! This is totally a bug in Chrome! If you download and print from [Chrome Canary](https://www.google.com/chrome/browser/canary.html), the pdf should look correct, backgrounds included.

**Comment by nilshg** [comment]
Ah, I was just going to comment that I've solved everything BUT the black printing issue, and that it works okay-ish in Firefox! Thanks a lot for the hint about Chrome Canary, works like a charm!

And thank you also for the extended answer to my earlier problem, I had never heard of Grunt before (plus I'm on Windows, plus on a work machine w/o admin rights), so the whole thing was getting a bit too advanced for me. Your suggestion about simply changing the reference to `reveal.css` worked - I recalled from previous versions that not using the minified css file would lead to terribly slow slide transitions, but that doesn't seem to be the case anymore.
